### PR TITLE
fixed test cases

### PR DESCRIPTION
**Pull request by egmox**
## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [ ] I have signed the [Adobe Open Source CLA](http://opensource.adobe.com/cla.html).
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes and the overall coverage did not decrease.
- [ ] All unit tests pass on CircleCi.
- [ ] I ran all tests locally and they pass.

### CODE REVIEW

File: ui.tests/test-module/specs/accordion/accordion.authoring.spec.js [hunk 1]
```diff
@@ -66,10 +66,14 @@ describe('Page - Authoring', function () {
         cy.get("[name='./custom']")
             .should("exist");
 
-        cy.get('.cq-dialog-cancel').should("exist").focus().click({ force: true }).then(() => {
-            cy.get('.cq-dialog-cancel').should("not.exist").then(() => {
-                cy.deleteComponentByPath(accordionDrop);
-            });
+        cy.get('.cq-dialog-cancel').should("exist").focus().click({ force: true });
```
Comment (sakshi-arora1) [inline ui.tests/test-module/specs/accordion/accordion.authoring.spec.js:69]: this line seems redundant now

File: ui.tests/test-module/specs/replaceaction/replacelayout-container.authoring.spec.js [hunk 2]
```diff
@@ -211,16 +219,14 @@ describe('component replace - Authoring', function () {
             cy.openAuthoring("/conf/core-components-examples/settings/wcm/templates/af-blank-v2/structure");
             updateAllowedComponent(true);
 
-            cy.openSiteAuthoring(replaceCompPageUrl);
+            cy.openSiteAuthoring(replaceCompPagePath);
```
Comment (sakshi-arora1) [inline ui.tests/test-module/specs/replaceaction/replacelayout-container.authoring.spec.js:222]: how is this different in cloud? I mean what is url in cloud vs 6.5

Comment (egmox) [reply ui.tests/test-module/specs/replaceaction/replacelayout-container.authoring.spec.js:222]: This is not different. We should change the variable name in master also as both variables have same value.

File: ui.tests/test-module/specs/radiobutton/radiobutton.authoring.spec.js [hunk 3]
```diff
@@ -146,7 +146,9 @@ describe('Page - Authoring', function () {
             cy.get("coral-selectlist-item").contains('Boolean').should('be.visible').click({force: true});
             cy.get(".cmp-adaptiveform-radiobutton__value").invoke('val', 'Not a Boolean');
             cy.get('.cq-dialog-submit').click();
-            cy.get('.coral-Form-errorlabel').should('contain.text', 'Value Type Mismatch');
+            cy.get('.cmp-adaptiveform-base__editdialogbasic [icon="alert"]').should('be.visible').click();
+            cy.get('.cmp-adaptiveform-base__editdialogbasic coral-tooltip').contains('Value Type Mismatch').should('be.visible');
+            // cy.get('.coral-Form-errorlabel').should('contain.text', 'Value Type Mismatch');
```
Comment (iamsudhanshu) [inline ui.tests/test-module/specs/radiobutton/radiobutton.authoring.spec.js:151]: Why original check is removed now?

Comment (iamsudhanshu) [inline ui.tests/test-module/specs/radiobutton/radiobutton.authoring.spec.js:151]: Can we remove the commented line

File: ui.tests/test-module/specs/replaceaction/replacelayout-container.authoring.spec.js [hunk 4]
```diff
@@ -177,13 +177,21 @@ describe('component replace - Authoring', function () {
...
-        const replaceCompPagePath = "/content/forms/af/core-components-it/samples/replace/replacewcmcomponents/basic",
-            replaceCompPageUrl = replaceCompPagePath;
+        const replaceCompPagePath = "/content/forms/af/core-components-it/samples/replace/replacewcmcomponents/basic";
 
         const updateAllowedComponent = function (allow) {
-            cy.get('[title="Adaptive Form Container [Root]"]').click()
+            cy.get('[title="Adaptive Form Container [Root]"]').trigger ('mouseover').then(() => {
+                cy.get('[title="Adaptive Form Container [Root]"]').click();
+            })
+                // .should('have.attr', 'title', 'Adaptive Form Container [Root]')
```
Comment (iamsudhanshu) [inline ui.tests/test-module/specs/replaceaction/replacelayout-container.authoring.spec.js:186]: remove commented lines

File: ui.tests/test-module/specs/replaceaction/replacelayout-container.authoring.spec.js (hunk 2 above)
Comment (iamsudhanshu) [reply ui.tests/test-module/specs/replaceaction/replacelayout-container.authoring.spec.js:222]: If value is same - why was it failing earlier?

File: ui.tests/test-module/specs/replaceaction/replacelayout-container.authoring.spec.js [hunk 5]
```diff
@@ -211,16 +219,14 @@ describe('component replace - Authoring', function () {
...
-            cy.openSiteAuthoring(replaceCompPageUrl);
+            cy.openSiteAuthoring(replaceCompPagePath);
             cy.selectLayer("Edit");
             cy.insertComponent(responsiveGridDropZoneSelector, "Adaptive Form Image", afConstants.components.forms.resourceType.formimage);
             cy.openEditableToolbar(sitesSelectors.overlays.overlay.component + replaceCompEditPathSelector);
             cy.invokeEditableAction("[data-action='replace']");
             const replacementComp = "[value='" + replaceCompTestGroup + "']";
             cy.get(replacementComp).click();
             cy.deleteComponentByPath(replaceCompDrop);
-            cy.openSiteAuthoring("/conf/core-components-examples/settings/wcm/templates/af-blank-v2/structure");
```
Comment (iamsudhanshu) [inline ui.tests/test-module/specs/replaceaction/replacelayout-container.authoring.spec.js:222]: why this is removed? We should be revalidating after removal - by reloading the page.

Comment (egmox) [reply ui.tests/test-module/specs/replaceaction/replacelayout-container.authoring.spec.js:222]: This is not validating anything, it only removes the test-group component from policies after validations are completed. However, I checked manually and cypress is clicking wrong element at this point.

**Review by sakshi-arora1 — APPROVED** [review]

**Pull request merged**